<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLogin</class>
 <widget class="QWidget" name="TelaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLabel" name="labelUsuario">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>15</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usu√°rio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUsuario">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSenha">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEntrar">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>116</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSenha">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>66</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Senha</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
